scripts/AxionPlotResonances.C

```diff
@@ -0,0 +1,81 @@
+int AxionPlotResonances() {
+    cout << "Hello world!" << endl;
+    TRestAxionField* ax = new TRestAxionField();
+    double ma_max = 0.1;
+    double Ea = 4.2;
+    int n_ma = 100000;
+    int vacio = 1;  // 1 if you want to include the vacuum probability, 0 if not
+    std::string gasName = "He";
+    std::pair<std::vector<double>, std::vector<double>> pair =
+        ax->GetMassDensityScanning(gasName, ma_max, Ea);
+    std::vector<double> photonMass = pair.first;
+    std::vector<double> density = pair.second;
+    std::vector<double> m_a;
+    std::vector<double> sum_prob;
+    double ma_step = (ma_max - 0.00025) / n_ma;

```

Comment:
why `0.00025`? What it is?

Reply:
It is the minimum value. Should be zero, but I was using that value because in the metadata the minimum energy for which we have the absorption length is that value.